specialized/async_specialized_form_parser.ipynb

```diff
@@ -0,0 +1,222 @@
+   },
+   "source": [
+    "# Document AI Specialized Parser (Async)\n",
+    "This notebook shows you how use Document AI's specialized parsers ex. Invoice, Receipt, and W9 asynchronously."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "_v0XtSwn7fmN",
+    "outputId": "373b9379-f6ac-451a-e428-ad9219fb31d0"
+   },
+   "outputs": [],
+   "source": [
+    "# Install necessary Python libraries\n",
+    "!pip install google-cloud-documentai\n",
+    "!pip install 'google-cloud-storage==1.31.0'\n",
+    "\n",
+    "!apt-get update"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Y8eO6Kcp7v2x"
+   },
+   "outputs": [],
+   "source": [
+    "from google.cloud import documentai_v1beta3 as documentai\n",
+    "from google.cloud import storage\n",
+    "\n",
+    "import re\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import simplejson as json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set your processor variables "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "k3c1mTa6IOk3"
+   },
+   "outputs": [],
+   "source": [
+    "PROJECT_ID = \"YOUR_PROJECT_ID_HERE\"\n",
+    "LOCATION = \"us\"  # Format is 'us' or 'eu'\n",
+    "PROCESSOR_ID = \"PROCESSOR_ID\"  # Create processor in Cloud Console\n",
+    "\n",
+    "GCS_INPUT_URI = 'gs://cesummit_workshop_data/invoice.pdf'\n",
+    "GCS_OUTPUT_URI = 'YOUR-OUTPUT-BUCKET'\n",
+    "GCS_OUTPUT_URI_PREFIX = 'TEST'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following code calls the synchronous API and parses the form fields and values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "hO3yJpDoJ3Zf"
+   },
+   "outputs": [],
+   "source": [
+    "def process_document_sample():    \n",
+    "    # Instantiates a client\n",
+    "    client_options = {\"api_endpoint\": \"{}-documentai.googleapis.com\".format(LOCATION)}\n",
+    "    client = documentai.DocumentProcessorServiceClient(client_options=client_options)\n",
+    "\n",
+    "    destination_uri = f\"{GCS_OUTPUT_URI}/{GCS_OUTPUT_URI_PREFIX}/\"\n",
+    "\n",
+    "    # 'mime_type' can be 'application/pdf', 'image/tiff',\n",
+    "    # and 'image/gif', or 'application/json'\n",
+    "    input_config = documentai.types.document_processor_service.BatchProcessRequest.BatchInputConfig(\n",
+    "        gcs_source=GCS_INPUT_URI, mime_type=\"application/pdf\"\n",
+    "    )\n",
+    "\n",
+    "    # Where to write results\n",
+    "    output_config = documentai.types.document_processor_service.BatchProcessRequest.BatchOutputConfig(\n",
+    "        gcs_destination=destination_uri\n",
+    "    )\n",
+    "\n",
+    "    # Location can be 'us' or 'eu'\n",
```

Comment:
Comment redundant. In the other notebooks. FWIW comment was:
    # The full resource name of the processor, e.g.:
    # projects/project-id/locations/location/processor/processor-id
    # You must create new processors in the Cloud Console first

Reply:
Great catch -- thank you, adding back the original comment

Reply:
Created gs://cloud-samples-data/documentai/async_invoices/ with 3, will add the GCS code to iterate through the folder and create the input configs tomorrow